library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacy.kt

```diff
@@ -0,0 +1,36 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay
+
+import com.datadog.android.sessionreplay.recorder.mapper.AllowAllWireframeMapper
+import com.datadog.android.sessionreplay.recorder.mapper.GenericWireframeMapper
+import com.datadog.android.sessionreplay.recorder.mapper.MaskAllWireframeMapper
+
+/**
+ * Defines the Session Replay privacy policy when recording the sessions.
+ * @see SessionReplayPrivacy.ALLOW_ALL
+ * @see SessionReplayPrivacy.MASK_ALL
+ *
+ */
+enum class SessionReplayPrivacy {
+    /** Does not apply any privacy rule on the recorded data. **/
+    ALLOW_ALL,
+
+    /**
+     *  Masks all the elements. All the characters in texts will be replaced by X, images will be
```

Comment:
Not a native speaker, but can we use plural form `texts` here? @fuzzybinary can you check?